cql3/statements/create_table_statement.cc

```diff
@@ -465,6 +465,20 @@ std::optional<sstring> check_restricted_table_properties(
                 throw exceptions::syntax_exception(fmt::format("The number of windows ({}) specified is larger than the allowed "
                                                    "number of windows specified by the twcs_max_window_count ({}) parameter.", window_count, max_windows));
             }
+        } else {
+              switch (qp.db().get_config().restrict_twcs_default_ttl()) {
+              case db::tri_mode_restriction_t::mode::TRUE:
+                  throw exceptions::configuration_exception(
+                      "TimeWindowCompactionStrategy tables without a strict default_time_to_live setting "
+                      " is forbidden. You may override this restriction by setting restrict_twcs_default_ttl "
+                      "configuration option to false.");
+              case db::tri_mode_restriction_t::mode::WARN:
+                  return format("TimeWindowCompactionStrategy tables without a default_time_to_live "
+                      "may potentially introduce too many windows. Ensure that insert statements specify a "
+                      "TTL (via USING TTL), when inserting data to this table");
```

Comment:
I suggest adding to this string also a phrase like "The ??? configuration option can be changed to silence this warning  or make it into an error".